lib/lynx/pipe/import.rb

```diff
@@ -8,7 +8,7 @@ def initialize(file)
       end
 
       def perform(command)
-        system("cat #{@file} | #{command}")
+        system("cat #{@file} | #{command}") || exit(1)
```

Comment:
Just a few thoughts.  You could consider raising or aborting here:

```
system(...) || abort('[lynx] Import pipe failed')
```

```
system(...) || raise Lynx::Error, 'Import pipe failed'
```

(in the second case you would have to create a `Lynx::Error` class:

```
module Lynx
  class Error < StandardError; end
end
```

But this would also allow callers to capture that error message if they wanted to.

Another option that might play well with the existing setup would be to introduce a `check` pipe that did something like:

```
def perform(command)
   @pipe.perform(command) || abort('[lynx] command failed with non-zero exit code')
end
```

That you could then combine to cause a 

```
Lynx.check(Lynx.append(...))
```

What I don't like about this is that you have to add a `check` call anywhere you want to check  command return status codes, and it seems like a reasonable default to fail if the command fails.

Probably my first preference would be to raise an error - that way, in our own scripts, we can rescue that error and provide a valuable error message.

Reply:
On second thought, I opted to wrap all systemouts in open3, and just raise aborts search for mysql and mysqldump. let me know if you have any thoughts on this

Reply:
on third thought, i decided to raise an error if systemouts fail, and will handle catching and aborting and cleanup in setup (in active)